apps/src/javalab/Javalab.js

```diff
@@ -287,7 +287,7 @@ Javalab.prototype.afterClearPuzzle = function() {
   project.autosave();
 };
 
-Javalab.prototype.onCommitCode = function() {
+Javalab.prototype.onCommitCode = function(filesToCommit, commitNotes) {
```

Comment:
do we still need these parameters?

Reply:
Wont we need them eventually? I know that method doesn't actually use them yet..

Reply:
oh we'll need the commit notes, but not filesToCommit

Reply:
🙌✅